src/metakb/transform/base.py

```diff
@@ -284,13 +293,13 @@ def _get_digest_for_str_lists(str_list: List[str]) -> str:
         :return: Digest
         """
         str_list.sort()
-        blob = json.dumps(str_list, separators=(",", ":")).encode("ascii")
+        blob = json.dumps(str_list, separators=(",", ":"), sort_keys=True).encode(
+            "ascii"
+        )
```

Comment:
When you squash + merge, I think this should be included in the commit message and not just descr since this is where the breaking change occurred

Reply:
Yeah I've been copying/pasting PR descriptions into the squash commits lately, since it also adds them to the local git history and doesn't just live on github

Reply:
We can change the default squash and merge settings to use PR title / description so less copy/pasta